### PR TITLE
add README translation

### PR DESCRIPTION
**Pull request by maelle**
Please review the Rmd file only :-)

### CODE REVIEW

File: README.es.md [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+
+Para ofrecerte para revisar paquetes, completa [este breve
+formulario](https://ropensci.org/software-reviewer/) proporcionando tu
+información de contacto y tus áreas de experiencia. Siempre buscamos más
+personas con experiencia en el desarrollo de paquetes en general y con
+conocimientos especializados en los campos en los que se utilizan los
+paquetes.
+
+Por favor también lee [nuestra guía para personas que realizan
+revisiones](https://devdevguide.netlify.app/es/softwarereview_reviewer.es.html).
```
Comment (yabellini) [inline README.es.md:91]:
```suggestion
revisiones](https://devguide.ropensci.org/es/softwarereview_reviewer.es.html).
```

File: README.es.md [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+
+## <a href="#further" name="further"></a>Más recursos
+
+Nuestra [Guía de
+desarrollo](https://devdevguide.netlify.app/es/index.es.html) también
```
Comment (yabellini) [inline README.es.md:96]:
```suggestion
desarrollo](https://devguide.ropensci.org/es/index.es.html) también
```

File: README.es.md [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+inlcuye:
+
+- [una guía para el equipo
+  editorial](https://devdevguide.netlify.app/es/softwarereview_editor.es.html)
```
Comment (yabellini) [inline README.es.md:100]:
```suggestion
  editorial](https://devguide.ropensci.org/es/softwarereview_editor.es.html)
```

File: README.es.md [hunk 4]
```diff
@@ -0,0 +1,440 @@
...
+
+- [nuestras
+  políticas](https://devdevguide.netlify.app/es/softwarereview_policies.es.html)
```
Comment (yabellini) [inline README.es.md:103]:
```suggestion
  políticas](https://devguide.ropensci.org/es/softwarereview_policies.es.html)
```

File: README.es.md [hunk 5]
```diff
@@ -0,0 +1,440 @@
...
+
+- capítulos dedicados a buenas prácticas para el mantenimiento de
+  paquetes
+
+- diferentes plantillas: [plantilla de
+  revisión](https://devdevguide.netlify.app/es/reviewtemplate.es.html),
```
Comment (yabellini) [inline README.es.md:109]:
```suggestion
  revisión](https://devguide.ropensci.org/es/reviewtemplate.es.html),
```

File: README.es.md [hunk 6]
```diff
@@ -0,0 +1,440 @@
...
+  [plantilla para quien hace la
+  edición](https://devdevguide.netlify.app/es/editortemplate.es.html),
```
Comment (yabellini) [inline README.es.md:111]:
```suggestion
  edición](https://devguide.ropensci.org/es/editortemplate.es.html),
```

File: README.es.md [hunk 7]
```diff
@@ -0,0 +1,440 @@
...
+  [plantilla para pedido de
+  revisión](https://devdevguide.netlify.app/es/reviewrequesttemplate.es.html).
```
Comment (yabellini) [inline README.es.md:113]:
```suggestion
  revisión](https://devguide.ropensci.org/es/reviewrequesttemplate.es.html).
```

File: README.es.md [hunk 8]
```diff
@@ -0,0 +1,440 @@
...
+
+# <a href="#editors" name="editors"></a>Equipo editorial y de revisión
+
+### Editores/as asociados/as
+
+El proceso de revisión por pares del software de rOpenSci está dirigido
+por:
+
+- [Noam Ross](https://github.com/noamross), EcoHealth Alliance, EEUU;
+- [Karthik Ram](https://github.com/karthik), rOpenSci, EEUU;
```
Comment (yabellini) [inline README.es.md:129]:
```suggestion
```

File: README.es.md [hunk 9]
```diff
@@ -0,0 +1,440 @@
...
+ropensci/software-review-meta](https://github.com/ropensci/software-review-meta/issues).
+
+# <a href="#editors" name="editors"></a>Equipo editorial y de revisión
+
+### Editores/as asociados/as
+
+El proceso de revisión por pares del software de rOpenSci está dirigido
+por:
+
+- [Noam Ross](https://github.com/noamross), EcoHealth Alliance, EEUU;
```
Comment (yabellini) [inline README.es.md:128]:
```suggestion
- [Mark [Padgham](https://github.com/mpadge)], rOpenSci, Germany;
- [Noam Ross](https://github.com/noamross), rOpenSci, EEUU;
```

File: README.es.md [hunk 10]
```diff
@@ -0,0 +1,440 @@
...
+- [Mauro Lepore](https://github.com/maurolepore), 2 Degrees Investing
+  Initiative, EEUU;
+- [Laura DeCicco](https://github.com/ldecicco-USGS), USGS, EEUU;
+- [Julia Gustavsen](https://github.com/jooolia), Agroscope, Suiza;
+- [Emily Riederer](https://github.com/emilyriederer), Capital One, EEUU;
+- [Adam Sparks](https://github.com/adamhsparks), Department of Primary
+  Industries and Regional Development;
+- [Jeff Hollister](https://github.com/jhollist), US Environmental
+  Protection Agency.
+
```
Comment (yabellini) [inline README.es.md:142]: I think we need to update this list. For example, Bea is not here.

File: README.es.md [hunk 11]
```diff
@@ -0,0 +1,440 @@
...
+  University of Science and Technology, Arabia Saudita;
+- [Leonardo Collado-Torres](https://github.com/lcolladotor), Lieber
+  Institute for Brain Development, EEUU;
+- [Toby Hocking](https://github.com/tdhock), Northern Arizona
+  University, EEUU.
+
+### Revisión y edición invitada
+
+Agradecemos a las siguientes personas, quienes ofrecieron su tiempo y
+experiencia para revisar paquetes enviaros a rOpenSci:
```
Comment (yabellini) [inline README.es.md:160]:
```suggestion
experiencia para revisar paquetes enviados a rOpenSci:
```

**Review by yabellini — APPROVED** [review]
Now that we have the dev guide in Spanish, I have reviewed this translation. The list of editors and reviewers needs to be updated. Is it a manual process? Is it a scheduled update?

File: README.es.md (hunk 10 above)
Comment (maelle) [reply README.es.md:142]: it will be knit again automatically but i'll check

**Pull request merged**